### ISSUE TITLE
reweight/iwan.py bug

### ISSUE THREAD

**Issue by sunray-bupt** [thuml/Transfer-Learning-Library#201]
**Describe the bug**
` def get_importance_weight(self, feature):
        """
        Get importance weights for each instance.
        Args:
            feature (tensor): feature from source domain, in shape :math:`(N, F)`
        Returns:
            instance weight in shape :math:`(N, 1)`
        """
        weight = 1. - self.discriminator(feature)
        weight = weight / weight.mean()
        weight = weight.detach()
        return weight`
`weight / weight.mean()`可能会产生除0操作，导致nan。

**To Reproduce**
增加条件判断
`if weight.mean(): weight = weight / weight.mean()`

**Comment by thucbx99** [comment]
感谢指出这个bug，这里除的时候增加一个epsilon就可以了，我们会尽快修一下～
